src/controllers/publish.ts

```diff
@@ -650,11 +680,19 @@ export const lookupReview = async (req: Request, res: Response, next: NextFuncti
             return;
         }
         let errors: ViewErrDTO | undefined;
+        const revisit = Boolean(dimension.factTableColumn !== dimension.metadata?.name);
 
         if (req.method === 'POST') {
             switch (req.body.confirm) {
                 case 'true':
-                    res.redirect(req.buildUrl(`/publish/${dataset.id}/dimension/${dimension.id}/name`, req.language));
+                    if (revisit)
+                        res.redirect(
+                            req.buildUrl(`/publish/${dataset.id}/dimension-data-chooser/${dimension.id}`, req.language)
+                        );
+                    else
+                        res.redirect(
+                            req.buildUrl(`/publish/${dataset.id}/dimension/${dimension.id}/name`, req.language)
+                        );
```

Comment:
Can we use curly brackets on conditions unless it's a 1-liner please

Reply:
It is a 1 liner... the issue here is that it hit the line limit so the ESline/Prettier changed the formatting... When your change goes in this will become a 1 line item.